gadget/gadget.go

```diff
@@ -80,6 +80,13 @@ type Volume struct {
 	Structure []VolumeStructure `yaml:"structure"`
 }
 
+func (v *Volume) EffectiveSchema() string {
+	if v.Schema == "" {
+		return "gpt"
```

Comment:
Wasn't there a constant for that?

Reply:
Ah, right. The commit was added almost a month ago, way before the consts were added. I'll update the PR accordingly.